moz_do/views.py

```diff
@@ -13,12 +14,17 @@
 @moz_do.route('/package/<pkgname>')
 def show_user_profile(pkgname):
     # show the package data for that package
-    return 'Package %s' % escape(pkgname)
+    return "Package %s" % (pkgname)
```

Comment:
We still want to escape the package name right?

Reply:
Yeah - though that's just debug until we get the actual model data.